src/index.js

```diff
@@ -45,7 +45,9 @@ import WorkflowPreview from "./app/views/workflow-preview/WorkflowPreview";
 import CreateContent from "./app/views/content/CreateContent";
 import NotFound from "./app/components/not-found";
 import "./scss/main.scss";
-
+import {errCodes} from "./app/utilities/errorCodes";
+import notifications from "./app/utilities/notifications";
+console.log('testing loading')
```

Comment:
May want to remove this line.

Not an issue but if removing the log line it may be worth adding back the spaces around the import braces on lines 8 and 48

Reply:
Removed the console log and added the spaces in.